README.md

```diff
@@ -172,6 +173,16 @@ Be aware of possible security threats and problems. You should at least be famil
 
 Good generic security guidelines would be:
 
+## Docker
+
+**Using Docker will not make your service more secure.** Generally, you should consider at least following things if using Docker:
+
+- Don't run any untrusted binaries inside Docker containers
+- Create unprivileged users inside Docker containers and run binaries using unprivileged user instead of root whenever possible
+- Periodically update the dependencies inside Docker containers
```

Comment:
Does this mean we would be advocating doing the equivalent of `apt-get upgrade` within a long-running container? Or does it mean that you should rebuild your images when there are security patches for any libraries or tools they contain?

Reply:
I think it's better practice to rebuild the containers, not to upgrade them on-the-fly. I'll clarify that aspect.

Reply:
Yeah, agree 👍